release_notes.md

```diff
@@ -1,5 +1,7 @@
 # Release Notes
 
+## Microsoft.Azure.Functions.Worker.Extensions.DurableTask v1.1.0-preview.1
```

Comment:
Should this be preview.2 or preview.3?

Reply:
I have it as `preview.1` because this is most likely what we will release publicly. What we have now is `entities-preview.N`, which is just for internal testing.